dilib-macros/src/provide_attr.rs

```diff
@@ -53,20 +53,22 @@ impl ProvideAttribute {
             .map(|s| Scope::from_str(&s))
             .unwrap_or(Scope::Scoped);
 
-        let bind = map.remove_entry(keys::BIND).map(|(_, value)| {
-            let type_string = value.to_string_literal().unwrap_or_else(|| {
+        let bind = map.remove_entry(keys::BIND).map(|(_, values)| {
+            let mut types = vec![];
+            for value in values.to_string_literal().unwrap_or_else(|| {
                 panic!(
                     "#[{}] '{}' must be a string literal",
                     keys::PROVIDE,
                     keys::BIND
                 )
-            });
-
-            // We need: Box<dyn TraitType + Send + Sync>
-            let boxed_type = format!("std::boxed::Box<dyn {} + Send + Sync>", type_string);
-
-            syn::parse_str::<Box<Type>>(&boxed_type)
-                .unwrap_or_else(|_| panic!("'{}' is not a valid trait type", type_string))
+            }).split(",") {
```

Comment:
Use a local variable for the split value string before iterating

Reply:
done

Reply:
@Neo-Ciber94 BTW I also sent an email to your hotmail 😄